docs/sphinx_guide/lists_tables.rst

```diff
@@ -236,11 +236,11 @@ Hlists
 .. hlist::
     :columns: 2
 
-    - .. figure:: images/yi_jing_01_chien.jpg
+    - .. figure:: ../images/yi_jing_01_chien.jpg
```

Comment:
If the `images` directory is going to be top level, you may want to simplify stuff with `/images` rather than `../images`?  I _think_ Sphinx treats that correctly when there's an output prefix and whatnot.